.github/workflows/int-and-connected-test-run.yml

```diff
@@ -1,103 +1,42 @@
 name: Unit, Smoke, Connected and Integration tests
-env:
-  K8_CLUSTER: 'integration-master'
-  # This must be defined for the bash redirection
-  GOOGLE_APPLICATION_CREDENTIALS: 'jade-dev-account.json'
-  # This must be defined for the bash redirection
-  GOOGLE_SA_CERT: 'jade-dev-account.pem'
-  # This will make test logs not use JSON
-  TDR_LOG_APPENDER: 'Console-Standard'
-  # The default Azure credentials to use to authenticate
-  AZURE_CREDENTIALS_APPLICATIONID: 0e29ec36-04e8-44d5-ae7c-50dc15135571
-  AZURE_CREDENTIALS_HOMETENANTID: fad90753-2022-4456-9b0a-c7e5b934e408
-  JADE_USER_EMAIL: connected-tdr-user@notarealemail.org
-  RBS_CLIENT_CREDENTIAL_FILE_PATH: rbs-tools-sa.json
-  AZURE_CREDENTIALS_SECRET: ${{ secrets.AZURE_CREDENTIALS_SECRET }}
-  AZURE_SYNAPSE_SQLADMINUSER: ${{ secrets.AZURE_SYNAPSE_SQLADMINUSER }}
-  AZURE_SYNAPSE_SQLADMINPASSWORD: ${{ secrets.AZURE_SYNAPSE_SQLADMINPASSWORD }}
-  AZURE_SYNAPSE_WORKSPACENAME: tdr-snps-int-east-us-ondemand.sql.azuresynapse.net
-  CACHE_PATHS: |
-    build/jacoco
-    build/reports
-    build/spotless
-    build/test-results
-    build/jacocoHtml
 on:
-  pull_request:
-    branches:
-      - develop
-    paths:
-      - '!*'
```

Comment:
Reviewer note: I think we should simplify the paths used to be all paths in a pull request.

Reply:
I have mixed thoughts on this. It makes sense that we wouldn't want to run this in certain cases, but if the action is marked as required, it puts PRs against those cases in limbo requiring admin override. For that reason, I favor leaving this out.

Reply:
Why change this now? If the idea is to convert and preserve behavior, I would move this change to a different PR so its impact can be evaluated separately.

Are there known problems with the current list of paths that changing it would fix? (E.g., a case where tests aren't being run but they should be?)

Also, if we do work on this part of the workflow action, it would be nice to address the issue where PRs that skip tests run require a force override when merging due to "pending" test results.

Reply:
> I think the current approach may also be related to why we're having the bug where we need to force override.

This is correct. I've run into this once where it was unclear why tests were not being run even though they are marked as required in the branch protection rules. It required a repo admin to override merge the PR.

Reply:
> I think the current approach may also be related to why we're having the bug where we need to force override.

The "bug" is in how this repo is configured. Changing this workflow to use include vs exclude shouldn't make a difference. The problem is that the repo says "Job X is required" but include/exclude config prevents the job X from running, so its status will never be reported. Although maybe having this config directly on the job, vs on a subworkflow, will change how this is reported.

>  I would prefer having a denylist using paths-ignore instead, which is the suggested mechanism

I don't see how the GitHub docs recommend `paths` vs `paths-ignore`; from my reading it describes both without specifying one as preferred over the other. It does say that `paths` is the most flexible as it allows both include and exclude path regex, where as `paths-ignore` doesn't support `!`.